.circleci/config.yml

```diff
@@ -1,10 +1,18 @@
-version: 2
-
-references:
-  container_config: &container_config
+version: 2.1
+executors:
```

Comment:
Before, `aeternity/infrastructure` was used. I assume it meant tag `latest` though I did not find this documented in Circle CI doc. Now tags are `master` and `stable`. `latest` tag depends on [integration tests in that repo to have passed](https://github.com/aeternity/infrastructure/blob/8fcad03dd84b0ae6cee6615be099d83637330b1f/.circleci/config.yml#L266-L268), `master` and `stable` do not . So tags used now are less stable than before. Is this intended?

Reply:
In description of ticket I read:
> For example integration should be deployed with aeternity/infrastructure:latest while UAT and main with aeternity/infrastructure:v1.7.0 (latest working for it)
>
> This will allow implementing BC breaking changes to deploy scripts in aeternity/infrastructure:master

That `latest` vs. tag name makes sense. Not sure about `master` vs. `stable`.

Reply:
`latest` is actually equal to `master`. However, it's true that `latest` is pushed after integration tests pass, while the `master` is pushed before that. This is actually downside of the workflow, but not bug. Otherwise we have to run integration tests on each branch which is expensive both time/cost.

So we definitely should use `latest` instead of `master` @michalbagrowski

The `stable` branch and docker tag are intended to be merged only from master with corresponding PR, thus its explicit that integration tests should have been passed for latest master.

Reply:
> The `stable` branch and docker tag are intended to be merged only from master with corresponding PR, thus its explicit that integration tests should have been passed for latest master.

Maybe worth placing a note in https://github.com/aeternity/infrastructure/wiki

Reply:
tracking in https://www.pivotaltracker.com/story/show/164457613